Dockerfile

```diff
@@ -1,10 +1,29 @@
+# Container image that runs your code
 FROM registry.access.redhat.com/ubi8/ubi
 
 ARG conftest_version=0.30.0
+ARG BATS_VERSION=1.6.0
+
+ENV POLICY_PATH="/project"
 
 RUN curl -L https://github.com/open-policy-agent/conftest/releases/download/v"${conftest_version}"/conftest_"${conftest_version}"_Linux_x86_64.tar.gz | tar -xz --no-same-owner -C /usr/bin/ && \
     dnf -y --setopt=tsflags=nodocs install \
     jq \
     skopeo
 
-COPY policies /project
\ No newline at end of file
+# Install BATS
+RUN curl -LO "https://github.com/bats-core/bats-core/archive/refs/tags/v$BATS_VERSION.tar.gz" && \
```

Comment:
Can we combine this RUN instruction with the one on the line 9 to prevent creating an extra layer in the image?

Reply:
Combined